batcher/aligned-batcher/src/zk_utils/mod.rs

```diff
@@ -65,21 +65,25 @@ fn verify_internal(verification_data: &VerificationData) -> bool {
             is_valid
         }
         ProvingSystemId::Risc0 => {
-            if let (Some(image_id_slice), Some(pub_input)) = (
-                &verification_data.vm_program_code,
-                &verification_data.pub_input,
-            ) {
-                let mut image_id = [0u8; 32];
-                image_id.copy_from_slice(image_id_slice.as_slice());
-                return verify_risc_zero_proof(
-                    verification_data.proof.as_slice(),
-                    &image_id,
-                    pub_input,
-                );
+            // Allow for empty public input
```

Comment:
This comment doesn't add much value here

Reply:
Addressed